environment/src/main/java/com/sequenceiq/environment/network/service/extended/SubnetType.java

```diff
@@ -0,0 +1,5 @@
+package com.sequenceiq.environment.network.service.extended;
+
+public enum SubnetType {
+    PUBLIC, MLX, DWX
```

Comment:
CB

Reply:
I used PRIVATE instead of CB but the same meaning